lms/templates/courseware/progress.html

```diff
@@ -80,6 +80,13 @@ <h2 class="title">${_("We're working on it...")}</h2>
                     <p class="copy">${_("We're creating your certificate. You can keep working in your courses and a link to it will appear here and on your Dashboard when it is ready.")}</p>
                   </div>
                   <div class="msg-actions"></div>
+                  %elif not is_id_verified or is_unverified:
+                   <div class="msg-content">
+                    ## Translators: This message appears to users when the users have not completed identity verification.
+                    <h2 class="title">${_("Certificate unavailable")}</h2>
+                    <p class="copy">${_("Your certificate is not issued because you do not have a current verified identity with edX. ")} <a href="${reverse('verify_student_reverify')}"> ${_("Verify your identity now.")}</a></p>
+                  </div>
+                  <div class="msg-actions"></div>
```

Comment:
I think this duplication is intended. 
The reason is that each template within the `if` `else` block has this div.msg-actions element, empty or not. If I leave this element out for my own conditional block, there would a style inconsistency. It's for keeping the styles to be the same.